cmd/juju/cloud/updatecredential.go

```diff
@@ -330,33 +330,26 @@ func (c *updateCredentialCommand) updateRemoteCredentials(ctx *cmd.Context, upda
 
 	var erred error
 	verified := map[string]jujucloud.Credential{}
+	mapUnion := func(new map[string]jujucloud.Credential) {
```

Comment:
Please use a better name other than "new".

Reply:
Sure, I'll change it to "item" :D